### DISCUSSION TITLE
Microsoft Live Share Support

### DISCUSSION THREAD

**Discussion by jefersonchaves** [forcedotcom/salesforcedx-vscode#5780] [Ideas]
**Feature Request**
Now more than even we are working remotely and many times we work in pairs. Would be very helpful to be able to push code and run the tests through test runner.

**Solution description**
The ability for the remote session to:
- Push code.
- View tests and to execute them.

**Alternatives**
Having a separate plugin to allow this to happen.
The current only workaround is to share a terminal between the sessions.

**Comment by git2gus** [comment]
This issue has been linked to a new work item: W-7618980

**Comment by git2gus** [comment]
This issue has been linked to a new work item: W-7618979

**Comment by smaddox-sf** [comment]
Hi @jefersonchaves - Thank you for submitting this idea.  I can see the usefulness and boost to productivity.  This isn't on our near-term roadmap but we'll consider it in the future.

**Comment by jefersonchaves** [comment]
Thanks @smaddox-sf, if there is something I could do to help, please don't hesitate to let me know.

**Comment by MGarfOppLoans** [comment]
Agreed, currently the biggest issue is you can't see deploy on save results while live sharing

**Comment by jefersonchaves** [comment]
Hello, @smaddox-sf, is there any chance that this can get in a roadmap from now on? It has been a while, and now most of us are remote.
